symbolic_pymc/tensorflow/printing.py

```diff
@@ -41,11 +47,50 @@ def println(self, obj):
         self.buffer.write(self.format(obj) + "\n")
         self.buffer.flush()
 
+    def depth_check(self):
+        if isinstance(self.depth_index, (list, tuple, range)):
+            return self.depth_count in self.depth_index
+        elif self.depth_index is None:
+            return True
+        else:
+            raise ValueError("Depth needs to be a tuple, list or range type.")
+
 
-def tf_dprint(obj, printer=None):
+def tf_dprint(obj, printer=None, depth_index=None):
     """Print a textual representation of a TF graph.
 
     The output roughly follows the format of `theano.printing.debugprint`.
+
+    with graph_mode():
```

Comment:
If you're going to add multi-line code examples in a docstring, we'll need to determine the proper formatting (e.g. RST for Sphinx rendering) and apply it universally.  When in doubt, follow PyMC3's conventions.